index.js

```diff
@@ -122,8 +123,23 @@ module.exports = (input, opts) => {
 		},
 		{
 			title: 'Bumping version using npm',
-			enabled: () => opts.yarn === false,
+			enabled: () => opts.yarn === false && !opts.contents,
 			task: () => exec('npm', ['version', input])
+		},
+		{
+			title: 'Bumping version using npm',
+			enabled: () => opts.yarn === false && opts.contents,
+			task: () => exec('npm', ['version', input], {cwd: path.join(process.cwd(), opts.contents)})
+		},
+		{
+			title: 'Commiting package changes',
```

Comment:
Doesn't it do this for you when using another `cwd`? If not, add these two steps inside an `if` statement like we do with `runPublish`.

```js
if (opts.contents) {
    tasks.add([...]);
}
```

Reply:
the `npm publish` task will only performs git related tasks if it finds a `.git` folder within the directory it is running within.